[jaeger-operator] Helm Linting error

``` 
 [ERROR] templates/crds.yaml: the kind "apiextensions.k8s.io/v1beta1 CustomResourceDefinition" is deprecated in favor of "apiextensions.k8s.io/v1 CustomResourceDefinition"
  Error: 1 chart(s) linted, 1 chart(s) failed
```

Seems it's time to replace api version

--- a/charts/jaeger-operator/Chart.yaml
+++ b/charts/jaeger-operator/Chart.yaml
@@ -1,11 +1,12 @@
 apiVersion: v1
 description: jaeger-operator Helm chart for Kubernetes
 name: jaeger-operator
-version: 2.18.0
+version: 2.18.1
 appVersion: 1.20.0
 home: https://www.jaegertracing.io/
 icon: https://www.jaegertracing.io/img/jaeger-icon-reverse-color.svg
-source: https://github.com/jaegertracing/jaeger-operator
+sources:
+  - https://github.com/jaegertracing/jaeger-operator
 maintainers:
   - email: ctadeu@gmail.com
     name: cpanato

--- a/charts/jaeger-operator/crds/crd.yaml
+++ b/charts/jaeger-operator/crds/crd.yaml
@@ -1,4 +1,4 @@
-apiVersion: apiextensions.k8s.io/v1beta1
+apiVersion: apiextensions.k8s.io/v1
 kind: CustomResourceDefinition
 metadata:
   name: jaegers.jaegertracing.io
@@ -31,3 +31,7 @@ spec:
     - name: v1
       served: true
       storage: true
+      schema:
+        openAPIV3Schema:
+          type: object
+          x-kubernetes-preserve-unknown-fields: true

--- a/ct.yaml
+++ b/ct.yaml
@@ -3,7 +3,7 @@ remote: origin
 chart-dirs:
   - charts
 chart-repos:
-  - incubator=https://kubernetes-charts-incubator.storage.googleapis.com
+  - incubator=https://charts.helm.sh/incubator
   - elastic=https://helm.elastic.co
   - bitnami=https://charts.bitnami.com/bitnami
 helm-extra-args: --timeout=600s

